CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [XOSTOR/Create] Add confirmation modal informing hosts toolstask will restart if packages need to be installed (PR [#7570](https://github.com/vatesfr/xen-orchestra/pull/7570))
```

Comment:
```suggestion
- [XOSTOR/Create] Add confirmation modal informing hosts toolstack will restart if packages need to be installed (PR [#7570](https://github.com/vatesfr/xen-orchestra/pull/7570))
```

Reply:
What do you think of the following?

IMHO, it is more user oriented.

```suggestion
- [XOSTOR] Require confirmation before creating SR because hosts toolstask will restart if packages need to be installed (PR [#7570](https://github.com/vatesfr/xen-orchestra/pull/7570))
```